.github/workflows/check-inactive-projects.yml

```diff
@@ -0,0 +1,80 @@
+# type: Project Management
+# owner: @camunda/monorepo-devops-team
+---
+name: Check for Inactive Projects
+
+on:
+  schedule:
+    # Run on the first Friday of every month at 9:00 AM UTC
+    - cron: '0 9 1-7 * 5'
+  workflow_dispatch:
+
+env:
+  GHA_BEST_PRACTICES_LINTER: enabled
+
+jobs:
+  check-inactive-projects:
+    permissions: {}
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    defaults:
+      run:
+        shell: bash
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Import Secrets
+        id: secrets
+        uses: hashicorp/vault-action@4c06c5ccf5c0761b6029f56cfb1dcf5565918a3b # v3.4.0
+        with:
+          url: ${{ secrets.VAULT_ADDR }}
+          method: approle
+          roleId: ${{ secrets.VAULT_ROLE_ID }}
+          secretId: ${{ secrets.VAULT_SECRET_ID }}
+          secrets: |
+            secret/data/products/camunda/ci/camunda CAMUNDA_GITHUB_ORG_PROJECT_TOKEN;
+            secret/data/products/camunda/ci/camunda SLACK_WEBHOOK_TEAM_COREAPPLICATION_LEADERSHIP;
+
+      - name: Check for inactive projects
+        id: check-projects
+        env:
+          GH_TOKEN: ${{ steps.secrets.outputs.CAMUNDA_GITHUB_ORG_PROJECT_TOKEN }}
+        shell: bash
+        run: |
+          .github/scripts/check-inactive-projects.sh
+
+      - name: Send Slack notification
+        if: steps.check-projects.outputs.inactive-projects-found != '0' && github.event_name != 'pull_request'
```

Comment:
the script should handle filtering for just issues and not pull requests, so this if condition can be simplified to not include the check for pull requests